packages/tags/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@zendeskgarden/react-tags",
+  "description": "Components relating to tags in the Garden Design System",
+  "author": "Zendesk Garden <garden@zendesk.com>",
+  "license": "UNLICENSED",
```

Comment:
Missing `"homepage": "https://garden.zendesk.com/"` here and change license to `Apache-2.0`.

Reply:
Updated. I was almost missing a few other fields that I've now added.